server/src/main/java/org/opensearch/action/admin/cluster/remotestore/stats/RemoteStoreStats.java

```diff
@@ -151,6 +208,38 @@ static final class Fields {
         static final String REMOTE_REFRESH_LATENCY_IN_MILLIS = "remote_refresh_latency_in_millis";
     }
 
+    static final class DownloadStatsFields {
+        /**
+         * Last successful segment download timestamp in milliseconds
+         */
+        static final String LAST_DOWNLOAD_TIMESTAMP = "last_download_timestamp";
+
+        /**
+         * Total number of segment files downloaded from the remote store for a specific shard
+         */
+        static final String TOTAL_SYNCS_FROM_REMOTE = "total_syncs_from_remote";
+
+        /**
+         * Total bytes of segment files downloaded from the remote store for a specific shard
+         */
+        static final String TOTAL_FILE_DOWNLOADS_IN_BYTES = "total_file_downloads_in_bytes";
```

Comment:
Lets remove the file word from the name and that way it would be consistent with upload naming as well.